code/SiteConfig.php

```diff
@@ -490,6 +456,15 @@ public function canEdit($member = null)
         return Permission::checkMember($member, "EDIT_SITECONFIG");
     }
 
+    public function canDelete($member = null)
+    {
+        $extended = $this->extendedCan(__FUNCTION__, $member);
+        if ($extended !== null) {
+            return $extended;
+        }
+        return false;
+    }
```

Comment:
Needed to prevent a delete button from being presented to admin users. No one should be able to delete `SiteConfig` records by default.